pc/rpc_client.hpp

```diff
@@ -474,61 +470,64 @@ namespace pc
     }
 
     template<class T>
-    size_t account_subscribe::get_data_val( T *res, size_t tlen ) const
+    size_t account_update::get_data_val( T *res, size_t tlen ) const
     {
       char *ptr = (char*)res;
       size_t len = get_rpc_client()->get_data_val( dptr_, dlen_, tlen, ptr );
       return len;
     }
 
+    // account data subscription
+    class account_subscribe : public account_update
+    {
+    public:
+      // parameters
+      void set_commitment( commitment );
+
+      account_subscribe();
+      void request( json_wtr& ) override;
+      void response( const jtree& ) override;
+      bool notify( const jtree& ) override;
+
+    private:
+      commitment  cmt_;
```

Comment:
Should `cmt_` be moved into the base class seeing all the derived classes have it?

Reply:
yes, thanks, I noticed that but didn't want to repeat the testing for a small change - will include in the next round